examples/ssl-factory/test/unit/org/apache/cassandra/security/KubernetesSecretsSslContextFactoryTest.java

```diff
@@ -149,18 +149,6 @@ public void buildTrustManagerFactoryHappyPath() throws IOException
         Assert.assertNotNull(trustManagerFactory);
     }
 
-    @Test(expected = IOException.class)
```

Comment:
This test was just failing because it was throwing different exception.

Would be great if somebody doublechecked this.

Reply:
even though I set keystore_password in `config` in this test, it still says that it is null but it should not ...

Reply:
I remember I fixed that one. Let me find out where did it go :)

Reply:
I found it! Since the failing test is about testing for Invalid Keystore file path, I added below lines in the test. Without those two lines, the new code generates `IllegalArgumentException` since Password is mandatory for file based keystores even before getting to read the keystore file path.

`
config.put(KEYSTORE_PASSWORD_ENV_VAR, "MY_KEYSTORE_PASSWORD");
config.put("MY_KEYSTORE_PASSWORD","ThisWontMatter");
`